### PR TITLE
Start and Stop scripts, fancy fonts

### PR DESCRIPTION
**Pull request by open-schnick**

### CODE REVIEW

File: init_setup.sh [hunk 1]
```diff
@@ -16,19 +16,25 @@ dbname="FileFighterDB"
...
 
 # Startup Message.
+
+echo "  _____   _   _          _____   _           _       _                 "
+echo " |  ___| (_) | |   ___  |  ___| (_)   __ _  | |__   | |_    ___   _ __ "
+echo " | |_    | | | |  / _ \ | |_    | |  / _\` | | '_ \  | __|  / _ \ | '__|"
+echo " |  _|   | | | | |  __/ |  _|   | | | (_| | | | | | | |_  |  __/ | |   "
+echo " |_|     |_| |_|  \___| |_|     |_|  \__, | |_| |_|  \__|  \___| |_|   "
+echo "                                     |___/                             "
+echo "                   Version 1.2 Last updated at 29.10.20                "
```
Comment (qvalentin) [inline init_setup.sh:30]: version could a variable maybe? So you don't have to change it in every file?

File: start_services.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# Check if docker is running
+if ! docker info >/dev/null 2>&1; then
+    echo "Docker is not running, install it or first and retry."
```
Comment (qvalentin) [inline start_services.sh:5]: is this even english?

File: start_services.sh [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+echo ""
+echo "Docker prerequisites matched. Docker instance running."
+echo "Stopping services..."
+
+docker start $restname
+docker start $frontendname
+docker start $dbname
+
+echo "Finished starting FileFighter services again."
+echo "You can stop them again with 'sudo ./stop_services'."
```
Comment (qvalentin) [inline start_services.sh:35]: sudo is not always required. But we can leave it like this

**Review by qvalentin — APPROVED** [review]
why is lib/remove_services.sh empty?

File: init_setup.sh (hunk 1 above)
Comment (open-schnick) [reply init_setup.sh:30]: so could date, yes.

File: start_services.sh (hunk 2 above)
Comment (open-schnick) [reply start_services.sh:5]: no :D

File: start_services.sh (hunk 3 above)
Comment (open-schnick) [reply start_services.sh:35]: yeah true, but docker sometimes needs sudo access.

**Pull request merged**